Casks/o/opencloud.rb

```diff
@@ -0,0 +1,39 @@
+cask "opencloud" do
+  arch arm: "arm64", intel: "x86_64"
+
+  version "1.0.0-rc.1"
+  sha256 arm:   "9b97142b7fa207cbb68e5a1926d9dafbd1375e06a5f2336b197f3fdb774a2626",
+         intel: "95443b0ea69d40d5916965aa7e3155bd10d048598a942fd9e78c519c1e1cdb7d"
+
+  url "https://github.com/opencloud-eu/desktop/releases/download/v#{version}/OpenCloud_Desktop-v#{version}-macos-clang-#{arch}.pkg"
+  name "OpenCloud Desktop"
+  desc "Desktop syncing client for OpenCloud"
+  homepage "https://github.com/opencloud-eu/desktop"
+
+  livecheck do
+    url "https://github.com/opencloud-eu/desktop/releases"
+    strategy :github_releases
+    regex(/^v?(\d+(?:\.\d+)+)$/i) # Only matches stable versions without suffixes
```

Comment:
```suggestion
  # TODO: Update this to use the `GithubLatest` strategy (without a regex or
  # `strategy` block) when a stable version becomes available.
  livecheck do
    url :url
    regex(/^v?(\d+(?:\.\d+)+.+)$/i)
    strategy :github_releases do |json, regex|
      json.filter_map do |release|
        next if release["draft"]

        match = release["tag_name"]&.match(regex)
        next if match.blank?

        match[1]
      end
    end
```

* This should use `url :url`, as the `GithubLatest` and `GithubReleases` strategies will convert the GitHub cask `url` to the relevant API URL (i.e., no need for a URL string here).
* `regex` should come before `strategy`.
* The cask is using `1.0.0-rc.1` as the `version`, so the regex needs to match the version and the suffix for now.
* The `GithubReleases` strategy skips draft and pre-release releases by default, so we have to use a `strategy` block to work with pre-release releases for now.